Dwolla.Client/DwollaClient.cs

```diff
@@ -50,6 +51,26 @@ public class DwollaClient : IDwollaClient
         public async Task<RestResponse<TRes>> PostAsync<TReq, TRes>(Uri uri, TReq content, Headers headers) =>
             await SendAsync<TRes>(CreatePostRequest(uri, content, headers));
 
+        public async Task<RestResponse> DeleteAsync<TReq>(Uri uri, TReq content, Headers headers) =>
+            await SendAsync(CreateDeleteRequest(uri, content, headers));
+
+        private async Task<RestResponse> SendAsync(HttpRequestMessage request)
+        {
+            var r = await _client.SendAsync(request);
+            if (r.Exception == null) return r;
+
+            var e = CreateException(r);

```

Comment:
You should be able to pull this into a method used here and by the other SendAsync